loadgen/bindings/c_api.cc

```diff
@@ -139,7 +139,7 @@ void DestroyQSL(void* qsl) {
 
 // mlperf::c::StartTest just forwards to mlperf::StartTest after doing the
 // proper cast.
-void StartTest(void* sut, void* qsl, const TestSettings& settings) {
+void StartTest(void* sut, void* qsl, TestSettings& settings) {
```

Comment:
Please keep this const and make a copy of the settings instead. From the caller's perspective, it'll be very unexpected for the settings they chose to change from one call to the next.